src/ios/CDVWKInAppBrowserUIDelegate.m

```diff
@@ -114,6 +114,25 @@ - (void)      webView:(WKWebView*)webView runJavaScriptTextInputPanelWithPrompt:
     [[self getViewController] presentViewController:alert animated:YES completion:nil];
 }
 
+- (WKWebView *)        webView:(WKWebView *)webView
+createWebViewWithConfiguration:(WKWebViewConfiguration *)configuration
+           forNavigationAction:(WKNavigationAction *)navigationAction
+                windowFeatures:(WKWindowFeatures *)windowFeatures
+{
+    if (navigationAction.request.URL) {
+        NSURL *url = navigationAction.request.URL;
+        NSString *urlPath = url.absoluteString;
+
+        if ([urlPath rangeOfString:@"https://"].location != NSNotFound || [urlPath rangeOfString:@"http://"].location != NSNotFound) {
+            [[UIApplication sharedApplication] openURL:url
+                                               options:@{}
+                                     completionHandler:^(BOOL success) {}];
+        }
```

Comment:
I gather the `else` case never really happens. But is it possible? If so do we need to handle it?

Reply:
We don't handle it. That's intentional. It's not clear how we would want to handle it. Since we return `nil` at the end, nothing will happen if the URL is not http/https.